README.md

```diff
@@ -98,6 +98,9 @@ A curated list of tools and resources for security incident response, aimed to h
 ### Linux Distributions
 * [Security Onion](https://github.com/Security-Onion-Solutions/security-onion) - Security Onion is a special Linux distro aimed at network security monitoring featuring advanced analysis tools.
 * [SIFT Workstation](http://digital-forensics.sans.org/community/downloads) - The SANS Investigative Forensic Toolkit (SIFT) Workstation demonstrates that advanced incident response capabilities and deep dive digital forensic techniques to intrusions can be accomplished using cutting-edge open-source tools that are freely available and frequently updated.
+* [CAINE](http://www.caine-live.net/index.html) - The Computer Aided INvestigative Environment (CAINE) contains numerous tools that help investigators during their analysis, including forensic evidence collection.
+* [DEFT](http://www.deftlinux.net/) - The Digital Evidence & Forensics Toolkit (DEFT) is a Linux distribution made for computer forensic evidence collection. It comes bundled with the Digital Advanced Response Toolkit (DART) for Windows.
+* [DEFT Zero](http://www.deftlinux.net/) - DEFT Zero is a light edition of DEFT, focused primarily on forensically sound evidence collection.
```

Comment:
I think it's a bit of an overkill to include both DEFT and DEFT Zero...
Don't you think?